Geocoder/src/main/java/com/example/Geocoder/CensusService.java

```diff
@@ -0,0 +1,54 @@
+package com.example.Geocoder;
+
+import java.net.URI;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.DefaultUriBuilderFactory;
+
+import reactor.core.publisher.Mono;
+
+@Service
+public class CensusService {
+
+    private final WebClient webClient;
+    private final DefaultUriBuilderFactory uriBuilderFactory;
+
+    //https://geocoding.geo.census.gov/geocoder/Geocoding_Services_API.pdf
+    //https://www.baeldung.com/spring-autowire
+
+    @Autowired //injecting the WebClient.Builder
+    public CensusService(WebClient.Builder webClientBuilder){
+        this.uriBuilderFactory = new DefaultUriBuilderFactory("https://geocoding.geo.census.gov");
+        this.webClient = webClientBuilder.uriBuilderFactory(uriBuilderFactory).build();
+    }
+
+    //A Container that emits 0 to 1 items. Mono allows for asynchronous development that improve efficieny, responsiveness 
+    //and resource utilization.
+    public Mono<String> submitAddress(Address address){
+
+        //https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/DefaultUriBuilderFactory.html
+        //https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/UriBuilder.html
```

Comment:
Suggestion: Are these comments needed? Could remove comments that are not talking about the functionality of the method or line of code. If you need these links as references, you can add them to the ReadMe file.